assets/schema_input.json

```diff
@@ -1,7 +1,7 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema",
-    "$id": "https://raw.githubusercontent.com/phac-nml/iridanextexample/main/assets/schema_input.json",
-    "title": "phac-nml/iridanextexample pipeline - params.input schema",
+    "$id": "https://raw.githubusercontent.com/phac-nml/clustersplitter/main/assets/schema_input.json",
```

Comment:
In this file, can you change the regex for matching an MLST file to `^\\S+\\.mlst(\\.subtyping)?\\.json(\\.gz)?$` similar to gasclustering: https://github.com/phac-nml/gasclustering/blob/main/assets/schema_input.json#L20

Reply:
Updated in e619f702d420330e9d8dee7ba17c035802f600f6